### PR TITLE
Add Docker Desktop 4.7.0 installer command line docs

### PR DESCRIPTION
**Pull request by mat007**
### Proposed changes

Added information about the upcoming new options for the Docker Desktop installer in 4.7.0.

I also fixed the whale icon in https://docs.docker.com/desktop/mac/install/ (search for «The Docker menu»).
I can make a separate PR just for this if needed.

### Unreleased project version (optional)

Do not merge until we release 4.7.0.

### CODE REVIEW

File: desktop/mac/install.md [hunk 1]
```diff
@@ -88,6 +90,18 @@ Your Mac must meet the following requirements to install Docker Desktop successf
 
     For more information, see [Docker Desktop License Agreement](../../subscription/index.md#docker-desktop-license-agreement). We recommend that you also read the [Blog](https://www.docker.com/blog/updating-product-subscriptions/){: target="_blank" rel="noopener" class="_" id="dkr_docs_desktop_install_btl"} and [FAQs](https://www.docker.com/pricing/faq){: target="_blank" rel="noopener" class="_" id="dkr_docs_desktop_install_btl"} to learn how companies using Docker Desktop may be affected.
 
+### Install from the command line
+
+After downloading `Docker.dmg`, to install Docker Desktop to the Applications folder, run for instance the following commands in a terminal:
+```
+sudo hdiutil attach Docker.dmg
+sudo /Volumes/Docker/Docker.app/Contents/MacOS/install --accept-license
```
Comment (stephen-turner) [inline desktop/mac/install.md:98]: I'm not sure we want to put the --accept-license in the example here. I understand why it's there (to show where the arguments go) but I don't want people copy- and pasting it and then saying that they never knowingly accepted the license.

File: desktop/mac/install.md [hunk 2]
```diff
@@ -88,6 +90,18 @@ Your Mac must meet the following requirements to install Docker Desktop successf
...
+sudo hdiutil detach /Volumes/Docker
+```
+
+The installer accepts the following flag:
+- `--accept-license`: disables prompting for license acceptance on Docker Desktop first run
```
Comment (stephen-turner) [inline desktop/mac/install.md:103]: Let's rephrase this to imply "you are now accepting the license" rather than "this disables having to accept the license".

`--accept-license`: accepts the license now, rather than requiring it to be accepted when the application is first run

Is there a copy of the license online that we can link to?

**Review by stephen-turner — CHANGES_REQUESTED** [review]

File: desktop/mac/install.md (hunk 1 above)
Comment (mat007) [reply desktop/mac/install.md:98]: Good point, I’ll remove the arguments from the example.

File: desktop/mac/install.md (hunk 2 above)
Comment (mat007) [reply desktop/mac/install.md:103]: > Is there a copy of the license online that we can link to?

Yes, there is https://www.docker.com/legal/docker-subscription-service-agreement/

**Comment by mat007** [comment]
Updated, can you take another look?

File: desktop/mac/install.md [hunk 3]
```diff
@@ -88,6 +90,19 @@ Your Mac must meet the following requirements to install Docker Desktop successf
 
     For more information, see [Docker Desktop License Agreement](../../subscription/index.md#docker-desktop-license-agreement). We recommend that you also read the [Blog](https://www.docker.com/blog/updating-product-subscriptions/){: target="_blank" rel="noopener" class="_" id="dkr_docs_desktop_install_btl"} and [FAQs](https://www.docker.com/pricing/faq){: target="_blank" rel="noopener" class="_" id="dkr_docs_desktop_install_btl"} to learn how companies using Docker Desktop may be affected.
 
+### Install from the command line
+
+After downloading `Docker.dmg`, to install Docker Desktop to the Applications folder, run for instance the following commands in a terminal:
```
Comment (usha-mandya) [inline desktop/mac/install.md:95]:
```suggestion
After downloading `Docker.dmg`, run the following commands in a terminal to install Docker Desktop in the Applications folder :
```

File: desktop/mac/install.md [hunk 4]
```diff
@@ -88,6 +90,19 @@ Your Mac must meet the following requirements to install Docker Desktop successf
...
+```
```
Comment (usha-mandya) [inline desktop/mac/install.md:96]: nit: add a blank line before the code

Comment (usha-mandya) [inline desktop/mac/install.md:96]:
```suggestion
```console
```

File: desktop/mac/install.md [hunk 5]
```diff
@@ -88,6 +90,19 @@ Your Mac must meet the following requirements to install Docker Desktop successf
...
+sudo hdiutil attach Docker.dmg
+sudo /Volumes/Docker/Docker.app/Contents/MacOS/install
+sudo hdiutil detach /Volumes/Docker
+```
+
+The `install` command accepts the following flag:
```
Comment (usha-mandya) [inline desktop/mac/install.md:102]:
```suggestion
The `install` command accepts the following flags:
```

File: desktop/mac/install.md [hunk 6]
```diff
@@ -88,6 +90,19 @@ Your Mac must meet the following requirements to install Docker Desktop successf
...
+- `--accept-license`: accepts the [Docker Subscription Service Agreement](https://www.docker.com/legal/docker-subscription-service-agreement) now, rather than requiring it to be accepted when the application is first run
```
Comment (usha-mandya) [inline desktop/mac/install.md:103]:
```suggestion
- `--accept-license`: accepts the [Docker Subscription Service Agreement](https://www.docker.com/legal/docker-subscription-service-agreement){: target="_blank" rel="noopener" class="_"} during installation, instead of prompting you to accept the license when the application is first run
```

File: desktop/mac/install.md [hunk 7]
```diff
@@ -88,6 +90,19 @@ Your Mac must meet the following requirements to install Docker Desktop successf
...
+- `--allowed-org=<org name>`: requires the user to sign in and be part of the specified Docker Hub organization when running the application
```
Comment (usha-mandya) [inline desktop/mac/install.md:104]:
```suggestion
- `--allowed-org=<org name>`: requires you to sign in and be part of the specified Docker Hub organization when running the application
```

File: desktop/windows/install.md [hunk 8]
```diff
@@ -102,14 +102,18 @@ Looking for information on using Windows containers?
...
+
 1. Double-click **Docker Desktop Installer.exe** to run the installer.
 
     If you haven't already downloaded the installer (`Docker Desktop Installer.exe`), you can get it from
     [**Docker Hub**](https://hub.docker.com/editions/community/docker-ce-desktop-windows/).
     It typically downloads to your `Downloads` folder, or you can run it from
     the recent downloads bar at the bottom of your web browser.
 
-2. When prompted, ensure the **Enable Hyper-V Windows Features** or the **Install required Windows components for WSL 2** option is selected on the Configuration page.
+2. When prompted, ensure the **Use WSL 2 instead of Hyper-V** option on the Configuration page is selected or not depending on your choice of backend.
```
Comment (usha-mandya) [inline desktop/windows/install.md:114]:
```suggestion
2. When prompted, ensure you've selected the right backend on the Configuration page. Select **Use WSL 2 instead of Hyper-V** only if your system supports this option.
```

File: desktop/mac/install.md (hunk 7 above)
Comment (mat007) [reply desktop/mac/install.md:104]: Well it’s very likely the user installing the software from the command line will be different than the user running it.
The command line installation is mainly targeted at company admins.

File: desktop/windows/install.md (hunk 8 above)
Comment (mat007) [reply desktop/windows/install.md:114]: If the system doesn’t support one of the options, it will not be available to the user.

File: desktop/mac/install.md (hunk 6 above)
Comment (mat007) [reply desktop/mac/install.md:103]: Here as well, the user running the application can be a different user than the «you» installing it.

File: desktop/mac/install.md (hunk 5 above)
Comment (stephen-turner) [inline desktop/mac/install.md:102]: flags

**Review by stephen-turner — CHANGES_REQUESTED** [review]
Looks good now, just one tiny correction

**Review by stephen-turner — APPROVED** [review]
Thanks for all the changes, LGTM now.

File: desktop/mac/install.md (hunk 7 above)
Comment (usha-mandya) [reply desktop/mac/install.md:105]: Sure, makes sense.

**Review by usha-mandya — APPROVED** [review]
LGTM
